packages/remix/src/generators/setup-tailwind/lib/update-remix-config.ts

```diff
@@ -0,0 +1,52 @@
+import { joinPathFragments, type Tree } from '@nx/devkit';
+import { tsquery } from '@phenomnomnominal/tsquery';
+
+export function updateRemixConfig(tree: Tree, projectRoot: string) {
+  const pathToRemixConfig = joinPathFragments(projectRoot, 'remix.config.js');
+
+  if (!tree.exists(pathToRemixConfig)) {
+    throw new Error(
+      `Could not find "${pathToRemixConfig}". Please ensure a "remix.config.js" exists at the root of your project.`
+    );
+  }
+
+  const fileContents = tree.read(pathToRemixConfig, 'utf-8');
+
+  const REMIX_CONFIG_OBJECT_SELECTOR =
+    'PropertyAccessExpression:has(Identifier[name=module], Identifier[name=exports])~ObjectLiteralExpression';
+  const ast = tsquery.ast(fileContents);
+
+  const nodes = tsquery(ast, REMIX_CONFIG_OBJECT_SELECTOR, {
+    visitAllChildren: true,
+  });
+  if (nodes.length === 0) {
+    throw new Error(`Remix Config is not valid, unable to update the file.`);
+  }
+
+  const configObjectNode = nodes[0];
+
+  const propertyNodes = tsquery(configObjectNode, 'PropertyAssignment', {
+    visitAllChildren: true,
+  });
+
+  for (const propertyNode of propertyNodes) {
+    const nodeText = propertyNode.getText();
+    if (nodeText.includes('tailwind') && nodeText.includes('true')) {
```

Comment:
issue: the `tailwind` flag was not stabilized until v1.16, and our plugin is still using `^1.15` for the Remix version. We should explicitly migrate to v1.16, or detect the version of Remix being used and switch to the previous unstable `future.unstable_tailwind` version of the flag when needed.

I would recommend just moving to 1.16.

Reply:
Created a PR to migrate remix to 1.16 https://github.com/nrwl/nx-labs/pull/283